task_spec/spec/acceptance/init_spec.rb

```diff
@@ -176,10 +180,10 @@ def bolt_on(target, type, action, object,
       expect(res["status"]).to eq("success")
     end
     # Some packages don't add this path
-    results = run_command_on('target', '/opt/puppetlabs/bin/puppet --version')
-    results.each do |res|
-      expect(res["status"]).to eq("success")
-    end
+    # results = run_command_on('target', '/opt/puppetlabs/bin/puppet --version')
```

Comment:
This test will fail on windows. Can we rely on the version task for verification of install?